### ISSUE TITLE
Does not work properly when the source code is using Lf as line endings

### ISSUE THREAD

**Issue by Philippe91** [Kir-Antipov/LineSorter#2]
In that case, sorting is (slightly) wrong. Converting to Cr/Lf solves the problem, but because of cross-platform development, I use Lf.
Very easy to reproduce: these 3 lines are not sorted.

a
c
b

Another thing: it seems that \Cr\Lf are sometimes added in place of \lf.

**Comment by Kira-NT** [comment]
Unfortunately, that's the way it looks...

Didn't think about it during development, so I'm sorry if my LineSorter messed up your files (I know how painful the different line endings in one file/project are for developers)!

It will be fixed in LineSorter 4.0!

**Comment by Kira-NT** [comment]
### Done!

Successfully added support for the following newline representations:

```
public enum NewlineType
{
    Undefined,
    LF,
    CRLF,
    CR,
    LFCR,
    // RS // If someone will need it - uncomment!
}
```

No more broken sortings and newline-type substitutions)

--------

This fix will be available at the [Marketplace](https://marketplace.visualstudio.com/items?itemName=KirAntipov.LineSorterByKir) immediately after the first update (*LineSorter 4.0*)!
